extension/src/lib/chromeEvented.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-env node */
+/* global fluid, require */
+
+"use strict";
+
+var gpii = fluid.registerNamespace("gpii");
+var chrome = chrome || require("sinon-chrome");
+
+fluid.defaults("gpii.chrome.eventedComponent", {
+    gradeNames: "fluid.component",
+    events: {
+        onCreate: null,
+        onTabOpened: null,
+        onTabUpdated: null
+    },
+    listeners: {
+        onCreate: {
+            funcName: "gpii.chrome.eventedComponent.init",
+            args: "{that}"
+        }
+    }
+});
+
+gpii.chrome.eventedComponent.init = function (that) {
+    chrome.tabs.onCreated.addListener(function (tab) {
+        that.events.onTabOpened.fire(tab);
+    });
+    chrome.tabs.onUpdated.addListener(function (tabId, changeInfo, oldTabStatus) {
+        that.events.onTabUpdated.fire(oldTabStatus);
```

Comment:
You may just want to pass along all of the arguments from the chrome event. The listener to infusion event can pick and choose what arguments it wants anyways, but this will make it more flexible.